Rule Botannia: British Bots for the Restaurant

EOBGames opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
Adds British Bots to the restaurant. Their imperial tastes span a vast selection of items, so be sure you're able to accomodate them!
![britbots](https://user-images.githubusercontent.com/58124831/110355397-76d39100-8039-11eb-8032-31fd8021f174.png)
Also, adds some food items specifically for the Britbots: Beef Wellington, Indian Chicken Curry, and the esteemed Full English breakfast.
Finally, adds a new telescreen type for the bar- useful for keeping tabs on your customers from the comfort of the kitchen. Will require mapping changes, which I'm considering as out of scope for this PR.
The directionals for the bots, and the Bobby outfit, were both made by @tralezab.
The sound was made by @Qustinnus.
In addition, this includes the changes from #57541.
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
Variety is the spice of life, and that goes double for the kitchen.
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:Inept, Tralezab, Qustinnus
add: The Robot Earth Tourism Program has expanded, bringing in the polite British-Bots, who are hungry for some food that reminds them of the Empire. Bot Appetit!
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

EOBGames committed:
britbot

EOBGames committed:
britbots: enhanced

Pugie commented:
I hope nobody asks me for a bloody cup o' tea...

Iamgoofball commented on code/modules/food_and_drinks/restaurant/customers/_customer.dm:
```diff
@@ -142,3 +142,20 @@
+	speech_sound = 'sound/creatures/tourist/tourist_talk_british.ogg'
+	clothing_sets = list("british_gentleman", "british_bobby")
+	orderable_objects = list(
+	/datum/venue/restaurant = list(/obj/item/food/fishandchips = 10, /obj/item/food/soup/stew = 10, , /obj/item/food/salad/ricepudding = 5, /obj/item/food/grilledcheese = 5, /obj/item/food/pie/meatpie = 5, /obj/item/food/benedict = 5, /obj/item/food/full_english = 2, /obj/item/food/soup/indian_curry = 3, /obj/item/food/beef_wellington/slice = 2),
```
remove the curry from this list, it's a known fact the british don't spice their food

Paxilmaniac commented:
I think you spelled it wrong, it's actually supposed to be "Bri'ish"

EOBGames committed:
bobbybot lines

MrMelbert commented:
![image](https://user-images.githubusercontent.com/51863163/110550934-3425bd80-80fa-11eb-8a75-cea9ee50267b.png)
![image](https://user-images.githubusercontent.com/51863163/110551081-79e28600-80fa-11eb-80ba-68bd01536b40.png)

manuel testmerge, round 157895, british bots appeared to have some wonky clothing.

Qustinnus commented:
lmao they got amerifat clothing

Qustinnus commented on code/modules/food_and_drinks/restaurant/_venue.dm:
```diff
@@ -5,7 +5,7 @@
 	var/max_guests = 6
 	///Weighted list of customer types
-	var/list/customer_types = list(/datum/customer_data/american = 5, /datum/customer_data/italian = 3, /datum/customer_data/french = 3, /datum/customer_data/japanese = 3, /datum/customer_data/japanese/salaryman = 2, /datum/customer_data/mexican = 10)
+	var/list/customer_types = list(/datum/customer_data/american = 5, /datum/customer_data/italian = 3, /datum/customer_data/french = 3, /datum/customer_data/japanese = 3, /datum/customer_data/japanese/salaryman = 2, /datum/customer_data/mexican = 3, /datum/customer_data/british = 4)
```
```suggestion
	var/list/customer_types = list(/datum/customer_data/american = 5, /datum/customer_data/italian = 3, /datum/customer_data/french = 3, /datum/customer_data/japanese = 3, /datum/customer_data/japanese/salaryman = 2, /datum/customer_data/mexican = 3, /datum/customer_data/british/bobby = 2, /datum/customer_data/british/gent = 2)
```

EOBGames committed:
I hate moths

they're all dusty and shit

EOBGames committed:
that should fix it

EOBGames committed:
grilled cheese, why have you forsaken me?

alton brown specifies that what we eat is not a grilled cheese sandwich, but instead a grilled sandwich containing melted cheese. in order to be a grilled cheese sandwich, the cheese itself must be grilled.

adamkad1 commented:
apparently some things dont pay up
